pyproject.toml

```diff
@@ -4,24 +4,28 @@ version = "0.0.1"
 description = "MLOps with Databricks"
 readme = "README.md"
 requires-python = ">=3.11"
-dependencies = ["lightgbm>=4.5.0, <5",
-                "scikit-learn>=1.5.1, <2",
-                "cloudpickle>=3.0.0, <4",
-                "catboost==1.2.7, <2",
-                "graphviz==0.20.3, <1",
-                "mlflow>=2.16.0, <3",
-                "numpy>=1.26.4, <2",
-                "pandas>=2.2.2, <3",
-                "pyarrow>=15.0.2, <16",
-                "cffi>=1.17.1, <2",
-                "scipy>=1.14.1, <2",
-                "matplotlib>=3.9.2, <4",
-                "databricks-feature-engineering>=0.6, <1"]
+dependencies = [
```

Comment:
very strict versioning is required if you use feature engineering package  and want to deploy an endpoint (including pandas and oter packages, also pyarrow must be fixed to 14.0.2 - default would be 15.x, and  model endpoint deployment would fail)